lib/t/news.ex

```diff
@@ -134,6 +134,51 @@ defmodule T.News do
             "size" => [375, 667]
           }
         ]
+      },
+      %{
+        id: 2,
+        story: [
+          %{
+            "background" => %{"color" => "#111010"},
+            "labels" => [
+              %{
+                "value" => dgettext("news", "Привет! 👋"),
+                "position" => [24.0, 80.0],
+                "background_fill" => "#F97EB9"
+              },
+              %{
+                "value" => dgettext("news", "Мы добавили\nактивную локацию."),
+                "position" => [24.0, 148.0],
+                "background_fill" => "#F97EB9"
+              },
+              %{
+                "value" =>
+                  dgettext(
+                    "news",
+                    "Теперь в ленте показывается\nпримерное расстояние\nдо пользователя."
+                  ),
+                "position" => [24.0, 238.0],
+                "background_fill" => "#F97EB9"
+              },
+              %{
+                "value" =>
+                  dgettext(
+                    "news",
+                    "Включи автоматическое\nопределение локации,\nчтобы она\nоставалась актуальной 👇"
+                  ),
+                "position" => [24.0, 356.0],
+                "background_fill" => "#F97EB9"
+              },
+              %{
+                "action" => "enable_auto_location",
+                "value" => dgettext("news", "Включить"),
```

Comment:
I'd call it `Enable autolocation` or something like that. It's important to be clear.